koku/api/resource_types/test/tests_views.py

```diff
@@ -56,16 +56,3 @@ def test_endpoint_view(self):
                 self.assertIsNotNone(json_result.get("data"))
                 self.assertIsInstance(json_result.get("data"), list)
                 self.assertTrue(len(json_result.get("data")) > 0)
-
-    def test_endpoints_invalid_query_param_still_returns(self):
```

Comment:
Why remove this test?

Reply:
I was unsure if it was necessary since the only additional thing it is testing is the added group by but I do not believe it makes sense to ever use a group by for this api. I am happy to return it if you think test would be worth it!

Reply:
Yes, these APIs aren't supporting a group_by or any query params, so totally good to remove this test.